source/common/config/discovery_grpc_stream.h

```diff
@@ -0,0 +1,133 @@
+#pragma once
+
+#include <functional>
+
+#include "envoy/grpc/async_client.h"
+
+#include "common/common/backoff_strategy.h"
+#include "common/common/token_bucket_impl.h"
+#include "common/config/utility.h"
+
+namespace Envoy {
+namespace Config {
+
+// Oversees communication for gRPC-using xDS: reestablishes the gRPC channel when necessary, and
```

Comment:
Nit "for gRPC xDS implementations (parent to both regular xDS and incremental xDS variants)."

Reply:
Nice, concise and more fully conveys the structure; thanks.